### PR TITLE
[feat] added onboarding question about previous adoptees

### PR DESCRIPTION
**Pull request by saitunguturu**
[//]: # "Feel free to customize this template and the emojis to your project's vibes"

### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

-added question about choosing previous adoptees

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

image

image

image

image

image

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

- test all possible answer choice combinations, make sure users cannot submit without answering all subparts

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

- need to rework the text input part for choosing number of current adoptees once we finalize the design

### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @InayaY3

### CODE REVIEW

File: components/Dropdown.tsx
Comment (InayaY3) [inline components/Dropdown.tsx]: created a dropdown component using your existing code. we'll eventually use this to replace the dropdown being used in the state question (which is unstyled) once we do more hi-fi styling + this makes the question file cleaner!

File: components/onboarding/OnboardingQuestionVeteran.tsx [hunk 1]
```diff
@@ -85,7 +84,7 @@ export default function OnboardingQuestionVeteran() {
           loadingClassName="text-white"
           loading={isSubmitting}
         >
-          Submit
+          Next
```
Comment (InayaY3) [inline components/onboarding/OnboardingQuestionVeteran.tsx]: you forgot to update the veteran question to show "next" instead of acting as the last onboarding question
